packages/spinners/lib/index.ts

```diff
@@ -0,0 +1,96 @@
+import Spinnies, { SpinnerOptions, StopAllStatus } from "spinnies";
+
+const spinnies = new Spinnies();
+
+/**
+ * Add a new spinner with the given name.
+ */
+export function add(
+  name: string,
+  options: SpinnerOptions | string
+): SpinnerOptions {
+  if (typeof options === "string") {
+    options = {
+      text: options
+    };
+  }
+  return spinnies.add(name, options);
```

Comment:
How does `spinnies` handle collisions?

Reply:
Good question - I'm 99% sure it just overwrites, as internally it just keeps each spinner config in a dictionary-like object.

Reply:
Yeah, per [the `spinnies` code](https://github.com/jcarpanelli/spinnies/blob/90e33ee6384eab6d4783a506301ab3dbaee24b00/index.js#L36-L51), it just overwrites.